test/src/unit-cppapi-deletes.cc

```diff
@@ -49,7 +50,198 @@
 using namespace tiledb;
 using namespace tiledb::test;
 
-struct DeletesFx {
+struct DeletesVFSFx {
```

Comment:
Can we make a base class for DeletesVFSFx and DeletesLocalFx to avoid any code duplication?

Reply:
I tried to do that to address your comment but it takes a lot of changes to the whole file and I don't thing all that re-work is worth it right now. We can open a follow up ticket if you think it's important.

Reply:
I have merged the 2 classes into one, enabling thus the delete tests on all VFSs. This uncovered a bug in vacuuming on remote VFSs tracked here: https://app.shortcut.com/tiledb-inc/story/46282/delete-tests-with-vacuuming-fail-on-remote-vfss-s3-etc